<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>821</width>
      <height>631</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Previous Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Is Leap</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Week Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Days Till Your Birthday</string>
     </property>
    </column>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="selectingSpinBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Date:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="birthdayBrowser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>BirthDay:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>710</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>From File in.txt</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="birthdayButton">
       <property name="text">
        <string>Add Birthday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="informationButton">
       <property name="text">
        <string>Get Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hideButton">
       <property name="text">
        <string>Hide Information</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>917</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Date</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
